### DISCUSSION TITLE
WEBSITE_LOAD_CERTIFICATES makes SSL work on Azure

### DISCUSSION THREAD

**Discussion by anders-marten** [ActiveLogin/ActiveLogin.Authentication#439] [General]
**Standalone.MvcSample** was working fine locally but did not work when deployed on Azure.
There was a problem with the SSL connection to BankID and I got the error: "**System.Net.Http.HttpRequestException: The SSL connection could not be established...**". 

After some searching and trying I found a solution: Add **WEBSITE_LOAD_CERTIFICATES** = * to the App Service configuration. 

**Here are some more reading about the issue:** 
https://azure.microsoft.com/en-us/blog/using-certificates-in-azure-websites-applications/ 
https://learn.microsoft.com/en-us/azure/app-service/configure-ssl-certificate-in-code 
https://stackoverflow.com/questions/48028690/ssl-tls-connections-not-working-when-deployed-as-azure-app-service

**Comment by PeterOrneholm** [comment]
Hi Anders!
Thanks for pointing this out! What version of Active Login are you using? Are we talking about the private certificate here?

We usually recommend loading the certificate from Azure KeyVault, and when doing so, this should not be an issue.

Found a blogpost on the topic (specifically for Active Login) with the same solution as yours :)
https://johan.driessen.se/posts/Loading-private-certificates-from-a-Dotnet-Core-app-in-an-Azure-Linux-App-Service/
